cl/phase1/network/services/BlsToExecutionChange.go

```diff
@@ -0,0 +1,110 @@
+package services
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/Giulio2002/bls"
+	libcommon "github.com/ledgerwatch/erigon-lib/common"
+	"github.com/ledgerwatch/erigon/cl/beacon/beaconevents"
+	"github.com/ledgerwatch/erigon/cl/beacon/synced_data"
+	"github.com/ledgerwatch/erigon/cl/clparams"
+	"github.com/ledgerwatch/erigon/cl/cltypes"
+	"github.com/ledgerwatch/erigon/cl/fork"
+	"github.com/ledgerwatch/erigon/cl/pool"
+	"github.com/ledgerwatch/erigon/cl/utils"
+)
+
+type blsToExecutionChangeService struct {
+	operationsPool    pool.OperationsPool
+	emitters          *beaconevents.Emitters
+	syncedDataManager *synced_data.SyncedDataManager
+	beaconCfg         *clparams.BeaconChainConfig
+}
+
+func NewBLSToExecutionChangeService(
+	operationsPool pool.OperationsPool,
+	emitters *beaconevents.Emitters,
+	syncedDataManager *synced_data.SyncedDataManager,
+	beaconCfg *clparams.BeaconChainConfig,
+) BLSToExecutionChangeService {
+	return &blsToExecutionChangeService{
+		operationsPool:    operationsPool,
+		emitters:          emitters,
+		syncedDataManager: syncedDataManager,
+		beaconCfg:         beaconCfg,
+	}
+}
+
+func (s *blsToExecutionChangeService) ProcessMessage(ctx context.Context, subnet *uint64, msg *cltypes.SignedBLSToExecutionChange) error {
+	// https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/p2p-interface.md#bls_to_execution_change
```

Comment:
As I almost finished, I found similar logic to process this msg here: https://github.com/ledgerwatch/erigon/blob/devel/cl/transition/impl/eth2/operations.go#L402
almost duplicate code